lib/services/import_user.rb

```diff
@@ -0,0 +1,36 @@
+class ImportUser
+  EXPECTED_COLUMNS = %w(first_name last_name email birth_date employment_date password)
+  STATUS_NEW  = 'NEW'
+  STATUS_OK   = 'OK'
+  STATUS_FAIL = 'FAIL'
+
+  attr_reader :error
+
+  def initialize(attributes, options = {})
+    @options = { verbose: false }.merge(options)
+    @attributes = attributes.select { |key| EXPECTED_COLUMNS.include?(key) }
```

Comment:
Should be regexp that tests data format (UA vs ISO) and convert it to appropriate.